onnxruntime/contrib_ops/cuda/bert/attention_softmax.h

```diff
@@ -39,7 +39,8 @@ __device__ inline void Softmax(const int all_sequence_length,
                                const int sequence_length,
                                const int valid_end,
                                const int valid_start,
-                               const T* add_before_softmax,
+                               const T* res_pos_bias,
```

Comment:
use full name relative_position_bias.
I am curious that, even if we use short name, it shall be rel_pos_bias, not res_pos_bias.

Reply:
good catch. it's typo